docs/modules/SDDs/pages/0022-managed-services-in-cluster.adoc

```diff
@@ -0,0 +1,111 @@
+= SDD 0022 - Managed Services in Cluster
+
+:sdd_author:    Tobias Brunner
+:sdd_owner:
+:sdd_reviewers:
+:sdd_date:      2020-06-19
+:sdd_status:    draft
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+Besides the system services Project Syn also provides additional services to the user of the platform. This SDD describes how some kind of managed services on Kubernetes with Project Syn can be achieved. It is considered to be the initial version of this concept.
+====
+
+== Motivation
+
+While the current state of Project Syn focuses on delivering system services to Kubernetes clusters, it should also take application services into consideration. There needs to be a way to install, configure and maintain application services and make them available to the user of the platform.
+
+As application services are considered services like databases, caches, queues and other similar applications. They are usually used by the end user application running on a Kubernetes cluster, but could also be used by applications running outside of the cluster (f.e. in another cluster or even a VM).
+
+This doesn't include services offered by the cloud.
+
+=== Goals
+
+* Offer application services to applications running in or outside of the cluster
+* Maintenance processes to maintain these application services, making them a managed service
+* Definition of how the end user application instantiates and consumes an application service
```

Comment:
Shouldn't monitoring/alerting also be a goal of this SDD?

Reply:
And maybe backup/restore?

Reply:
Yes, I'll add that...